### PR TITLE
Fix search forms

### PR DESCRIPTION
**Pull request by samuelechu**
Fixes regressions introduced by https://github.com/connexta/revelio/pull/270.

- Add header section back into search forms editor
- fix console errors

image

### CODE REVIEW

File: src/main/webapp/components/query-builder/query-builder.tsx [hunk 1]
```diff
@@ -85,6 +85,11 @@ const QueryBuilder = (props: QueryBuilderProps) => {
     'Result Form': addResultForm,
   }
 
+  const AddOptions = () =>
+    addOptionsRef
+      ? ReactDOM.createPortal(<AddButton options={options} />, addOptionsRef)
+      : null
+
```
Comment (samuelechu) [inline src/main/webapp/components/query-builder/query-builder.tsx]: Fixes 
image

File: src/main/webapp/components/search-forms/editor.tsx [hunk 2]
```diff
@@ -22,6 +25,31 @@ if (typeof window !== 'undefined') {
...
+  onChange: (query: QueryType) => void
+}
+
+const Header = (props: HeaderProps) => {
+  const { query = {} } = props
+  return (
+    <Box display="flex" style={{ padding: 8 }} alignItems="center">
+      <TextField
+        fullWidth
+        value={query.title || ''}
```
Comment (samuelechu) [inline src/main/webapp/components/search-forms/editor.tsx]: fixes 
image
https://stackoverflow.com/a/47012342

File: src/main/webapp/components/search-forms/editor.tsx [hunk 3]
```diff
@@ -112,8 +143,18 @@ export const QueryEditor = (props: EditorProps) => {
       </Typography>
       <Divider />
       <Box width="100%" display="flex" flexDirection="row" height="100%">
-        <Box style={{ height: `calc(100% - 60px)`, width: 500 }}>
-          <QueryBuilder query={query} onChange={props.onChange} />
+        <Box style={{ height: `calc(100% - 134px)`, width: 500 }}>
```
Comment (samuelechu) [inline src/main/webapp/components/search-forms/editor.tsx]: dunno if there's a better way to do this, I just adjusted the pixels until the vertical scrollbar disappeared from the editor

Comment (willwill96) [reply src/main/webapp/components/search-forms/editor.tsx]: Does this look okay on both 1080p and 4k?

Comment (samuelechu) [reply src/main/webapp/components/search-forms/editor.tsx]: Yep :) Have to zoom out a bit to get the scroll bars to disappear in 1080p

**Pull request merged**